test/logger/POSTLogger.test.ts

```diff
@@ -102,31 +89,21 @@ describe('MeetingSessionPOSTLogger', () => {
 
   describe('logging level', () => {
     it('should log info with LogLevel.INFO and ignore the debug', done => {
-      const logger = new MeetingSessionPOSTLogger(
-        'testLogger',
-        configuration,
-        batchSize,
-        intervalMs,
-        BASE_URL,
-        LogLevel.INFO
-      );
+      const logger = new POSTLogger(batchSize, intervalMs, BASE_URL, LogLevel.INFO);
       logger.info('info');
       logger.error('error');
+      logger.debug('error');
       expect(logger.getLogCaptureSize()).is.equal(2);
       logger.stop();
       done();
```

Comment:
Do we need done for this test?

Reply:
so I removed the `intervalScheduler` and used `wait` and not using `done` pattern anymore.